test/functional/test_framework/address.py

```diff
@@ -41,7 +42,26 @@ def byte_to_base58(b, version):
         str = str[2:]
     return result
 
-# TODO: def base58_decode
+def base58_decode(s):
```

Comment:
This is currently unused in the test code, so I don't see the value of adding it unless this is going to be used any time soon


 I'd say to just remove the TODO

Reply:
It seems like it could easily be used to test the byte_to_base58 function, helping find & debug problems related to it